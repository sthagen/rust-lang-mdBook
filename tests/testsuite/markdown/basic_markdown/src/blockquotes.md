# Blockquotes

Empty:
>

Normal:
> foo
> bar

Contains code block:
> ```rust
> let x = 1;
> ```

Random stuff:
> ### And now,
>
> **Let us _introduce_**
> All kinds of
>
> - tags
> - etc
> - stuff
>
> 1. In
> 2. The
> 3. blockquote
>
>    > cause we can
>    >
>    > > Cause we can
