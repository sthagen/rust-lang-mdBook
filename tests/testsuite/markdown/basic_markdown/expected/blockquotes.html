<h1 id="blockquotes"><a class="header" href="#blockquotes">Blockquotes</a></h1>
<p>Empty:</p>
<blockquote></blockquote>
<p>Normal:</p>
<blockquote>
<p>foo
bar</p>
</blockquote>
<p>Contains code block:</p>
<blockquote>
<pre class="playground"><code class="language-rust"><span class="boring">#![allow(unused)]
</span><span class="boring">fn main() {
</span>let x = 1;
<span class="boring">}</span></code></pre>
</blockquote>
<p>Random stuff:</p>
<blockquote>
<h3 id="and-now"><a class="header" href="#and-now">And now,</a></h3>
<p><strong>Let us <em>introduce</em></strong>
All kinds of</p>
<ul>
<li>tags</li>
<li>etc</li>
<li>stuff</li>
</ul>
<ol>
<li>
<p>In</p>
</li>
<li>
<p>The</p>
</li>
<li>
<p>blockquote</p>
<blockquote>
<p>cause we can</p>
<blockquote>
<p>Cause we can</p>
</blockquote>
</blockquote>
</li>
</ol>
</blockquote>